mods/ra/rules/defaults.yaml

```diff
@@ -892,6 +892,8 @@
 	MapEditorData:
 		ExcludeTilesets: INTERIOR
 		Categories: Tree
+	RequiresSpecificOwners:
+		ValidOwnerNames: Neutral
```

Comment:
We'll also want this on the burnt tree husks, and maybe other actors.  We have mpspawn as a testcase so it might be worthwhile splitting the "normal" rules for this into a followup PR where we can bikeshed all the extra actor types that should have this?